usaspending_api/submissions/management/commands/rm_submission.py

```diff
@@ -37,6 +39,12 @@ def signal_handler(signal, frame):
         except ObjectDoesNotExist:
             raise RuntimeError(f"Broker submission id {submission_id} does not exist")
 
+        # Mark associated Accounts as updated, so they will be reloaded in ES nightly load
+        for faba in FinancialAccountsByAwards.objects.filter(submission_id=submission_id).all():
+            if faba.award:
+                faba.award.update_date = datetime.now(timezone.utc)
+                faba.award.save()
+
```

Comment:
Same thing here.  You could do this in a bulk statement along the lines of `Award.objects.filter(id__in=FinancialAccountsByAwards.objects.filter(submission_id=submission_id).values("award_id")).update(update_date=datetime.now(timezone.utc))` (untested).

Reply:
Oooh... so close... missing the `.values("award_id")`... very important... otherwise it'll try to delete awards based on faba ids.